Units/BaseUnit.java

```diff
@@ -0,0 +1,17 @@
+package Units;
+
+public class BaseUnit {
```

Comment:
Needs to change into an interface. Also make subinterfaces for mage, range, etc.